myco_api_client/setups/myco_matcher.py

```diff
@@ -1,15 +1,22 @@
 import logging
+import os
 from time import sleep
 
-from d3a_interface.matching_algorithms import PayAsBidMatchingAlgorithm
-
+from myco_api_client.matchers import RedisBaseMatcher
 from myco_api_client.matchers.base_matcher import BaseMatcher
+from myco_api_client.matching_algorithms import AttributedMatchingAlgorithm
+
+if os.environ["MYCO_CLIENT_RUN_ON_REDIS"] == "true":
+    base_matcher = RedisBaseMatcher
+else:
+    base_matcher = BaseMatcher
 
 
-class MycoMatcher(BaseMatcher):
+class MycoMatcher(base_matcher):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.is_finished = False
+        self.errors = 0
```

Comment:
I know that this is helping the integration tests. However, if I understand correctly this class is supposed to be the base class for non-GSy users of the Myco client, in which case this member variable is not that useful. Therefore I would recommend to remove it from here and either create a base class that only the integration tests inherit from that includes it, or implement it in each individual test. I would favor for the former.

Reply:
You are right, actually it's not used here so I can remove it (in the `TestRedisMycoMatcher` we have this field though)